### DISCUSSION TITLE
3.0.0-beta06

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#120] [Announcements]
- 修复弹幕在某些速度倍率下会闪烁的问题
- 增加手势调音量的灵敏度
- 增加自动版本更新检查
- 修复一些机型上无法登录的问题
- 弹幕自动简繁转换 (根据系统设置)
- 一些 UI 细节优化
- 播放器增加发送弹幕的 UI (功能还没实现)
  - 长期征集发弹幕广告词
   ![image](https://github.com/Him188/ani/assets/12100985/fd04c05f-ecd8-4d14-8da4-ea454ad2c946)
- 桌面端: 修复打开剧集列表会崩溃的问题

----

**附件:**

- `.zip` 是包含 JRE 的解压即用的绿色版本；
- `.dmg`、`.deb`、`.rpm` 和 `.msi` 分别是 macOS、Debian、RedHat 和 Windows 平台的安装程序；
- `.exe` 是包含 JRE 的 Windows 程序，可以直接运行。
- `.jar` 是不包含 JRE 的最小程序，需要配合 JRE 11 及以上使用；
- `.apk` 是 Android 应用，支持的最低 Android 版本为 8.0；

注意，桌面平台仅支持 64 位操作系统。在使用 macOS 的安装程序时可能提示文件损坏，这是因为此版本应用没有签名，可以忽略错误继续使用。

也可扫描以下二维码下载 Android 版本：
![](https://github.com/Him188/ani/releases/download/v3.0.0-beta06/ani-3.0.0-beta06.apk.qrcode.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Him188/ani/releases/tag/v3.0.0-beta06'>3.0.0-beta06</a>.</em>

**Comment by NeKoOuO** [comment]
“發個彈幕炒熱氣氛！”
“來個彈幕吧！”
“發個友善的彈幕吧！”
“是不是忍不住想發彈幕了呢？”

**Reply by Him188** [reply]
已加入随机列表
目前就先简繁混合吧, 过段时间做一下i18n

**Reply by NeKoOuO** [reply]
> 已加入随机列表
> 目前就先简繁混合吧, 过段时间做一下i18n

移strings感覺就很辛苦

**Reply by NeKoOuO** [reply]
> 已加入随机列表
> 目前就先简繁混合吧, 过段时间做一下i18n

簡繁混合是指...？🤔

**Reply by Him188** [reply]
你写的内容我以繁体加进去了
而我 (GPT) 写的内容是简体

**Reply by NeKoOuO** [reply]
> 你写的内容我以繁体加进去了
> 而我 (GPT) 写的内容是简体

用gpt做廣告詞嗎？

**Reply by Him188** [reply]
目前的广告词只有两条是我写的, 其他都是 GPT 的

**Reply by Him188** [reply]
之后打算用 https://paratranz.cn/projects 做本地化翻译

**Reply by NeKoOuO** [reply]
> 之后打算用 https://paratranz.cn/projects 做本地化翻译

哇塞這什麼酷東西

**Reply by NeKoOuO** [reply]
> 之后打算用 https://paratranz.cn/projects 做本地化翻译

這是要本地化什麼啊🤔

裡面不都是遊戲嗎？

**Reply by Him188** [reply]
翻译 Ani 的繁体中文, 如果未来有别的语言的用户也能方便地翻译英文

**Reply by NeKoOuO** [reply]
> 翻译 Ani 的繁体中文, 如果未来有别的语言的用户也能方便地翻译英文
> 

如果要翻譯的話 大部分是直接暴力轉換 所以產生很多一簡對多繁的異體字

所以我覺得用[OpenCC](https://github.com/BYVoid/OpenCC) 是個不錯的選擇

根據詞彙進行本地化翻譯

**Reply by Him188** [reply]
OpenCC 好东西, 未来可以用来转换弹幕

ParaTranz 是用来人工翻译的, 当我替换完字符串成翻译文本的引用之后就可以导出到 ParaTranz 去给社区人工翻译各种其他语言. 这样比较长久, 我只熟悉简体中文和英文, 其他的语言就只能靠社区了, 如果用翻译软件然后直接写项目里未来就比较难扩展
到时候做翻译再看看

**Comment by NeKoOuO** [comment]
「一起加入熱鬧的彈幕氛圍吧！」  
「一則友善的彈幕，讓氣氛更加溫暖！」  
「忍不了了！我要發彈幕！！」  
「是時候加入彈幕的行列，一起創造歡樂的氛圍！」
「用彈幕連結彼此的快樂！」  
「一起打造一個充滿歡笑和溫暖的環境！」
